src/svyhist.jl

```diff
@@ -0,0 +1,25 @@
+sturges(n::Integer) = ceil(Int, log2(n)) + 1
+
+sturges(vec::AbstractVector) = ceil(Int, log2(length(vec))) + 1
+
+sturges(df::DataFrame, var::Symbol) = ceil(Int, log2(size(df[!, var], 1))) + 1
+
+sturges(design::svydesign, var::Symbol) = sturges(design.variables, var)
+
+
+"""
+Histogram plot of a survey design variable.
+"""
+
```

Comment:
There should be no space between the documentation and the function. Line 13 should be removed.

Reply:
Furthermore, we need an example, just like we have in `svyby`, `svyglm`, etc